packages/ng-schematics/src/component/index.ts

```diff
@@ -67,7 +67,11 @@ export function singleComponent(templateOptions: TemplateOptions, skipRoute: boo
 }
 export function component(options: ComponentOptions): Rule {
 	return async (tree: Tree, context: SchematicContext) => {
-		App.initialize();
+		App.initialize("angular-cli");
+		GoogleAnalytics.post({
+			t: "screenview",
+			cd: "Add"
+		});
```

Comment:
At the end, when all custom options are resolved we should call GA again with something like this:
```
			GoogleAnalytics.post({
				t: "event",
				ec: "$ig add", - we may call it here with $ng add
				ea: `template id: ${argv.template}; file name: ${argv.name}`,
				cd1: selectedTemplate.framework,
				cd2: selectedTemplate.projectType,
				cd5: selectedTemplate.controlGroup,
				cd7: selectedTemplate.id,
				cd8: selectedTemplate.name,
				cd11: !!config.skipGit,
				cd14: config.project.theme
			});
```